ui/app/styles/components/lifecycle-chart.scss

```diff
@@ -0,0 +1,114 @@
+.lifecycle-chart {
+  padding-top: 2rem;
+  position: relative;
+
+  .lifecycle-phases {
+    position: absolute;
+    top: 1.5em;
+    bottom: 1.5em;
+    right: 1.5em;
+    left: 1.5em;
+  }
+
+  .lifecycle-phase {
+    position: absolute;
+    bottom: 0;
+    top: 0;
+
+    border-top: 2px solid transparent;
+
+    .name {
+      padding: 0.5rem 0.9rem;
+      font-size: $size-7;
+      font-weight: $weight-semibold;
+      color: $ui-gray-500;
+    }
+
+    &.is-active {
+      background: $white-bis;
+      border-top: 2px solid $vagrant-blue;
+
+      .name {
+        color: $vagrant-blue;
+      }
+    }
+
+    &.prestart {
+      left: 0;
+      right: 75%;
+
+      // Adapted from https://kovart.github.io/dashed-border-generator/
+      background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3cline x1='100%25' y1='0' x2='100%25' y2='100%25' fill='none' stroke='#{encode-color($ui-gray-200)}' stroke-width='3' stroke-dasharray='1%2c 7' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
```

Comment:
I’m sad to say it didn’t even occur to me to put the SVG in the template, I guess because I did originally start with `1px dashed` and was therefore too CSS-focused. I’ve just pushed a commit that adds a `divider` element to the template but styles it with CSS, which permits `stroke: $ui-gray-200`! Much better, thanks for prompting this.

Reply:
Ha, and I forgot you can still style the svg with sass. Good thing we were both thinking about it.